client/src/components/Card/Card.Body.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Would advise against using this case style word.word.extension, might be problematic for some operating systems, I didn't find any problems on a first search but I still believe it would be better to stick to a more common case style like PascalCase or camelCase.
The MOST used case style in the industry for react/react native projects is to name a file using PascalCase in case it is a file that exports a component (like this one) and camelCase should be used for helper files.
**This requires changes.**